bin/postgres_common/postgres/pgbackrest_info.sh

```diff
@@ -13,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-source /opt/cpm/bin/pgbackrest_env.sh > /tmp/pgbackrest_env.stdout 2> /tmp/pgbackrest_env.stderr
+CRUNCHY_DIR=${CRUNCHY_DIR:-'/opt/crunchy'}
+source $CRUNCHY_DIR/bin/postgres/pgbackrest_env.sh > /tmp/pgbackrest_env.stdout 2> /tmp/pgbackrest_env.stderr
```

Comment:
```suggestion
source "${CRUNCHY_DIR}/bin/postgres/pgbackrest_env.sh" > /tmp/pgbackrest_env.stdout 2> /tmp/pgbackrest_env.stderr
```

Please ensure environmental variables are properly quote throughout. I'm using this as one example.